src/commands/Utility/spotify.js

```diff
@@ -1,95 +1,136 @@
+const RichEmbed = require('discord.js').RichEmbed;
+const SpotifyWebApi = require('spotify-web-api-node');
+const SpotifyUriParser = require('spotify-uri');
 const got = require('got');
+const spotifyApi = new SpotifyWebApi();
+
+exports.run = async function (bot, msg, args) {
+    await got('https://spotify-auth.doxylamin.pw/').then((response) => {
```

Comment:
The only concern I have with this is that the auth may go down, which is bad, of course.
But I think you'll do your best to keep it up at least.